### ISSUE TITLE
Redirection vs. Location (Trac #61)

### ISSUE THREAD

**Issue by mnot** [httpwg/httpbis-issues#61]
The definition of the Location header currently says:
                    
"The Location response-header field is used to redirect the recipient to a location other than the Request-URI for completion of the request or identification of a new resource. (...)"
                    
The issue is that this can be read as if the "...or identification..." can be read as applying to the "redirect" part.
                   

Migrated from https://trac.ietf.org/ticket/61
```json
{
    "status": "closed",
    "changetime": "2008-04-03T11:56:25",
    "_ts": "1207223785000000",
    "description": "The definition of the Location header currently says:\n                    \n\"The Location response-header field is used to redirect the recipient to a location other than the Request-URI for completion of the request or identification of a new resource. (...)\"\n                    \nThe issue is that this can be read as if the \"...or identification...\" can be read as applying to the \"redirect\" part.\n                   ",
    "reporter": "mnot@pobox.com",
    "cc": "",
    "resolution": "fixed",
    "time": "2007-12-22T02:43:43",
    "component": "p2-semantics",
    "summary": "Redirection vs. Location",
    "priority": "",
    "keywords": "",
    "milestone": "unassigned",
    "owner": "",
    "type": "editorial",
    "severity": ""
}
```

**Comment by mnot** [comment]
Trac update at 20071222T02:44:03: mnot@pobox.com commented:

> One simple fix seems to be to swap the two parts of the sentence, as in...:
>                 
> "The Location response-header field is used for the identification of a new resource or to redirect the recipient to a location other than the Request-URI for completion of the request. (...)"
> 
>

**Comment by mnot** [comment]
Trac update at 20080104T06:06:15: 
* mnot@pobox.com changed component from "" to "messaging"
* mnot@pobox.com changed milestone from "" to "unassigned"
* mnot@pobox.com changed owner from "" to ""

**Comment by mnot** [comment]
Trac update at 20080104T06:28:31: 
* mnot@pobox.com changed component from "messaging" to "semantics"
* mnot@pobox.com changed owner from "" to ""

**Comment by mnot** [comment]
Trac update at 20080403T01:04:43: mnot@pobox.com changed type from "design" to "editorial"

**Comment by mnot** [comment]
Trac update at 20080403T11:56:25: 
* julian.reschke@gmx.de commented:

> Fixed in [237]:
> 
> Resolve #61: change the order of statements to make it clearer that "identification of a new resource" is not a case of redirection (closes #61).


* julian.reschke@gmx.de changed resolution from "" to "fixed"
* julian.reschke@gmx.de changed status from "new" to "closed"
